src/animation/animation.h

```diff
@@ -59,6 +59,7 @@ class LUMIX_ANIMATION_API Animation : public Resource
 		Vec3* m_positions;
 		Quat* m_rotations;
 		uint32_t* m_bones;
+		int m_fps;
```

Comment:
it's used once as an integer and once as a float, so it does not matter from performance view